capstone-frontend/src/BookSearchTile.test.js

```diff
@@ -42,16 +42,22 @@ test('renders book search tile test', () => {
   const bookListButton = document.getElementById("dropdown-list-add");
   expect(bookListButton).toBeInTheDocument();
   expect(bookListButton.innerHTML).toEqual("Add to List");
-  
 
+  await act(async () => {bookListButton.click()});
+  const dropdownItems = document.getElementsByClassName("dropdown-item");
+  expect(dropdownItems.length).toEqual(3);
+  expect(dropdownItems[0].innerHTML).toEqual("<span>Best Books</span>");
 });
 
-test('renders book search tile with empty bookList test', () => {
+test('renders book search tile with empty bookList test', async () => {
   render(<BookSearchTile book={book}
-    userBookLists={emptyArray} />, container);
+    bookLists={emptyArray} />, container);
 
   const bookListButton = document.getElementById("dropdown-list-add");
   expect(bookListButton).toBeInTheDocument();
   expect(bookListButton.innerHTML).toEqual("No Lists Found");
 
+  await act(async () => {bookListButton.click()});
```

Comment:
Same here: see if it still works when omitting `async`/`await`

Reply:
This second case was removed since the other conditional button was made a regular button instead of a dropdown button requiring a click to 'activate'